### PR TITLE
[FR] History of osu!

### PR DESCRIPTION
**Pull request by Airwam**
French translation of osu! wiki - History of osu! page.

### CODE REVIEW

File: wiki/History_of_osu!/fr.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+- [2007](/wiki/History_of_osu!/2007) - Débuts du jeu
+- [2008](/wiki/History_of_osu!/2008) - Taiko and Catch the Beat, plus de mods, et améliorations majeures
+- [2009](/wiki/History_of_osu!/2009) - osu! iPhone
+- 2010
+- [2011](/wiki/History_of_osu!/2011) - osu! wiki
+- [2012](/wiki/History_of_osu!/2012) - Cinq ans d'osu!, osu!mania, prise en charge des grands écrans, et paramètres visuels
+- [2013](/wiki/History_of_osu!/2013) - Refonte de l'interface utilisateur, skins en HD, (plus) de supports pour les grands écrans, éditeur osu!mania, collections en jeu, moddingV2 introduit, Disqus, osu!api, affichage de la précision, Comic Fiesta 2013, téléchargements illimités\*, et osu!cuttingedge (seulement pour les osu!supporters)
+- [2014](/wiki/History_of_osu!/2014) - scrolling par le BPM, séquences d'intro/outro en jeu, osu!cuttingedge (pour tout le monde\*), et la neige!
+- [2015](/wiki/History_of_osu!/2015) - jizz (osu!web repensé), osu!weekly, and osu!coins
+- [2016](/wiki/History_of_osu!/2016) - scorev2, osu! open-source annoncé (osu!lazer), et osu! wiki (refonte)
```
Comment (DoryanR) [inline wiki/History_of_osu!/fr.md:18]:
```suggestion
- [2016](/wiki/History_of_osu!/2016) - scorev2, osu! open-source annoncé (osu!lazer), et osu!wiki (refonte)
```

**Review by DoryanR — CHANGES_REQUESTED** [review]
After changes, it will be good for me.

File: wiki/History_of_osu!/fr.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+- [2007](/wiki/History_of_osu!/2007) - Débuts du jeu
+- [2008](/wiki/History_of_osu!/2008) - Taiko et Catch the Beat, plus de mods, et améliorations majeures
+- [2009](/wiki/History_of_osu!/2009) - osu! iPhone
+- 2010
+- [2011](/wiki/History_of_osu!/2011) - osu!wiki
+- [2012](/wiki/History_of_osu!/2012) - Cinq ans d'osu!, osu!mania, prise en charge des grands écrans, et paramètres visuels
+- [2013](/wiki/History_of_osu!/2013) - Refonte de l'interface utilisateur, skins en HD, (plus) de supports pour les grands écrans, éditeur osu!mania, collections en jeu, moddingV2 introduit, Disqus, osu!api, affichage de la précision, Comic Fiesta 2013, téléchargements illimités\*, et osu!cuttingedge (seulement pour les osu!supporters)
+- [2014](/wiki/History_of_osu!/2014) - scrolling par le BPM, séquences d'intro/outro en jeu, osu!cuttingedge (pour tout le monde\*), et la neige !
+- [2015](/wiki/History_of_osu!/2015) - jizz (osu!web repensé), osu!weekly, and osu!coins
```
Comment (DoryanR) [inline wiki/History_of_osu!/fr.md:17]:
```suggestion
- [2015](/wiki/History_of_osu!/2015) - jizz (osu!web repensé), osu!weekly, et osu!coins
```

File: wiki/History_of_osu!/fr.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+- [2016](/wiki/History_of_osu!/2016) - scorev2, osu! open-source annoncé (osu!lazer), et osu!wiki (refonte)
+- [2017](/wiki/History_of_osu!/2017) - Sortie du premier client d'osu!lazer, et intégration Discord
+- [2018](/wiki/History_of_osu!/2018) - ?
+
+---
+
+A noter que le osu! wiki a un peu de son histoire détaillée éparpillée à travers les pages.
```
Comment (DoryanR) [inline wiki/History_of_osu!/fr.md:24]:
```suggestion
A noter que le osu!wiki a un peu de son histoire détaillée éparpillée à travers les pages.
```

**Review by DoryanR — CHANGES_REQUESTED** [review]
Still some things to work on.

**Review by DoryanR — APPROVED** [review]
After checking the file, I approve the changes.

**Review by TicClick — APPROVED** [review]

**Pull request merged**